### ISSUE TITLE
Feature Request: option to add IMDb Rating or Genre to .png .ico or .jpg

### ISSUE THREAD

**Issue by DKPC69** [ramdany7/RightClick-Folder-Icon-Tools#11]
Hey ramdany7 I'm the guy who recently commented on your YouTube channel about how amazing this script is! it's the best thing I've ever come across, I was just wandering if you would ever implement a feature in which you could add IMDb info or/ and genre to an existing picture in one of the 3 formats (sortov like a "update IMDb or update genre" option to ".ico, .png .jpg" in the style of the plex plugin kometa ![image](https://github.com/user-attachments/assets/ab813941-4eef-46d4-ae70-cd01e3e635de) https://kometa.wiki/ it would be such a good feature to have, I see you already have the feature for IMDb rating and genre option for the cases but I haven't figured out how to get it to work on the 1. (None).bat but I think having the option to add these as overlays to existing pictures would be an amazing update, thanks again for this amazing tool "(if you ever need help with logos I can always try my best to make them for you), cheers

**Comment by ramdany7** [comment]
Oh yeah, hi! Thanks for the star—it’s nice when someone recognizes my work. Could you provide a sample with before-and-after pictures? If you’re using Photoshop, I might need the .psd file so I can reference anything necessary for the ImageMagick.

for example:
![sample](https://github.com/user-attachments/assets/4659cb4e-8baa-4327-a22d-b7fb1997c2c3)
I might need sample for the .nfo file too if you want it to show that "percentage" rating and the logo on the upper left cause mine doesnt provide that details inside it.

**Comment by DKPC69** [comment]
What you have in that example is perfect and exactly what I was looking for I just noticed media manager dosent export available streaming services for to the .nfo files ("kometa might use the built in plex stream check function") not sure if it does rotten tomatoes ratings either, but yeah that would be awesome to get the feature in your example if it isn't too much of a pain

**Comment by ramdany7** [comment]
yeah.. I can't tell which logo should it use if there wasn't details inside the .nfo file, and so the rotten tomatoes rating.. 

Here’s what I’ve come up with:
![Screenshot_518](https://github.com/user-attachments/assets/3f24714c-71e1-4cd8-a288-abca1540d4e7)

you can download the template here: [Simple template.zip](https://github.com/user-attachments/files/16216311/Simple.template.zip)

this template will work only on a potrait images, any landscape image will be strect out to match the "simple-shape.png".
just incase you want to reshape it, here is the psd template i create to made this template: [simple psd.zip](https://github.com/user-attachments/files/16216309/simple.psd.zip)

feel free to ask me if you need anything.

**Comment by DKPC69** [comment]
you are the best man i really appreciate it, this is perfect going to try it now, thankyou

**Comment by DKPC69** [comment]
Hey ramdany7 sorry to be a pain but i tested that simple template out and its getting "themoviedb ratings instead of the imdb ratings ive attached the picture and the nfo ("had to change .nfo to .log so i could post it on here"). I have tried for hours to change it but im too much of an amateur when it comes to these types of bat files

themoviedatabase rating is 7.3 and imdb is 7.9 

[Anthony Jeselnik Caligula (2013).log](https://github.com/user-attachments/files/16227768/Anthony.Jeselnik.Caligula.2013.log)
![Anthony Jeselnik Caligula (2013)-poster_1080p](https://github.com/user-attachments/assets/7e0a84a6-1cda-4b03-9ffc-2edd088f8ecb)

**Comment by ramdany7** [comment]
Hi, sorry for the late reply.

Here's the edited template with options to change the preferred rating:
[Simple template (imdb rating).zip](https://github.com/user-attachments/files/16232491/Simple.template.imdb.rating.zip)
There are 3 options: IMDB, TMDB, and TVDB. 
If there is no value matching the selected option, it will use whatever value is inside the .nfo file.

I'm also planning to edit all templates so they will load the .nfo details through `\resources\Extract-NFO_file.bat` instead of being embedded inside each template.

**Comment by DKPC69** [comment]
Your a legend thankyou, will be good to learn off that to see what was done I personally tried multiple times then I tried with 4 different ai platforms to try edit it to no avail haha have a good night man 👍

**Comment by DKPC69** [comment]
Oh I forgot to tell you I posted about your project on Reddit a couple of days ago hope you don’t mind here’s the link https://www.reddit.com/r/DataHoarder/s/DVKaqBQ8dq

**Comment by ramdany7** [comment]
> Oh I forgot to tell you I posted about your project on Reddit a couple of days ago hope you don’t mind here’s the link https://www.reddit.com/r/DataHoarder/s/DVKaqBQ8dq

Oh yeah.. that's fine. 
About that, I was actually quite surprised cause my repo got so much stars and many downloads all of the sudden (normally it doesn't do that).  So, I thought someone might have shared this project somewhere, and so I search it up and found your post. Only I dont really use reddit since it's blocked and restricted in my country.

But yeah, it's cool. I’m glad someone finds this project helpful, though it’s a bit embarrassing; my code is super messy and I don’t really know how to write code properly. 😅
